components/net_traits/lib.rs

```diff
@@ -137,6 +137,7 @@ pub struct LoadData {
     pub data: Option<Vec<u8>>,
     pub cors: Option<ResourceCORSData>,
     pub pipeline_id: Option<PipelineId>,
+    pub include_third_party_credentials: bool,
```

Comment:
Yes, you're right. Let's call this `credentials_flag` and link to `// https://fetch.spec.whatwg.org/#concept-http-fetch step 4.3`